lib/output.c

```diff
@@ -2642,6 +2654,21 @@ grn_output_result_set_close_v1(grn_ctx *ctx,
   grn_output_array_close(ctx, outbuf, output_type);
 }
 
+static void
+grn_output_table_records_content_v3(grn_ctx *ctx,
+                                    grn_obj *outbuf,
+                                    grn_content_type output_type,
+                                    grn_obj *table,
+                                    grn_obj_format *format)
+{
+  grn_output_table_records_content(ctx, outbuf, output_type, table, format);
+  if (format) {
+    grn_output_table_records_close(ctx, outbuf, output_type);
+  } else {
+    grn_output_array_close(ctx, outbuf, output_type);
+  }
```

Comment:
Do we need them?
We want to call them in `grn_output_table_records_close_v3()`.

Reply:
Ah, it will be done in the next PR?

Reply:
I added `grn_output_table_records_close_v*` because it can be used in the next PR.